packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -851,6 +851,112 @@ void main() {
     expect(tester.getTopLeft(find.text('Item 0')).dx, 72);
     expect(tester.getTopLeft(find.text('Item 1')).dx, 72);
   });
+
+  testWidgets('test tooltip', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Column(
+            children: <Widget>[
+              // Using three different PopupMenuButton's to test
```

Comment:
Instead of this one comment, maybe have three separate comments to describe the three different test situations where it is relevant